test/unit/lib/ToolbarPanelHub.test.js

```diff
@@ -1,13 +1,31 @@
 import { _ToolbarPanelHub } from "lib/ToolbarPanelHub.jsm";
 import { GlobalOverrider } from "test/unit/utils";
 
+const FAKE_MESSAGE = {
```

Comment:
Can we import this from `OnboardingProvider`? Easier to keep in sync & validate schema this way.